README.md

```diff
@@ -1 +1,25 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## A Little bit about me.
+
+Hello my name is Arturo Juarez. I am studying **International Relations** under the Committee of International Relations at the *University of Chicago*. I was born in California but I grew up in both Illinios and and Southern California. As such I consider myself both a Midwesterner and a Californian. My interests incluce reading, outdoor running, and traveling to new destinations both domestically and internationally. 
+
+### More about me
+
+My *hobbies* include playing sports such as basketball, baseball, and volleyball. I enjoy the **outdoors** as much as I enjoy spending a lazy day **indoors**. I care deeply about the social, economic, and civil challenges facing our democracy today and I actively donate my time, energy, and resources to many causes I find of pressing importance. I *source* my news at times from [link](https://www.npr.org/) [link](https://www.washingtonpost.com/) or [link](https://thehill.com/) 
+
+#### Lists
+
+* Unordered List
+* Red Fruit
+  + strawberries 
+  + apples 
+  + raspberries 
+  
+  1. Ordered List
+  2. Green Fruit
+    + avocado 
+    + grapes 
+    + pears
+
+image: ![](IMG_6652.JPG)
```

Comment:
It looks like the image didn't show up in the rendered version. I also had a really hard time getting the image feature to work and actually posted about it on the class GitHub--it turns out you have to upload any images to your GitHub in order to have them appear on things you put here. This is because when you're online GitHub can't display pictures if they're only saved on your computer.